### ISSUE TITLE
MacOS HomeBrew installing the previous PowerShell Core version.

### ISSUE THREAD

**Issue by MaximoTrinidad** [PowerShell/PowerShell#6703]
I finally got my MacOS Sierra VM installed but to my surprise, following the installation instruction, HomeBrew is only installing PowerShell Core version 6.0.2. 

Steps to reproduce
------------------
Follow Github installation instructions: https://github.com/PowerShell/PowerShell/tree/master/docs/installation/macos.md

Expected behavior
-----------------
To install the latest version: 6.1.0-preview.1

Actual behavior
---------------
Installs previous version: 6.0.2

Environment data
----------------

```powershell
PS /Users/maxt> $PSVersionTable                                                                     

Name                           Value                                                               
----                           -----                                                               
PSVersion                      6.0.2                                                               
PSEdition                      Core                                                                
GitCommitId                    v6.0.2                                                              
OS                             Darwin 17.5.0 Darwin Kernel Version 17.5.0: Mon Mar  5 22:24:32 P...
Platform                       Unix                                                                
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                             
PSRemotingProtocolVersion      2.3                                                                 
SerializationVersion           1.1.0.1                                                             
WSManStackVersion              3.0                                                                 

```
![macos_pwsh_2018-04-23_10-01-38](https://user-images.githubusercontent.com/19892435/39131465-77386762-46dd-11e8-82a5-40c2f89f485e.jpg)

**Comment by MaximoTrinidad** [comment]
Awesome! I appreciate the information. Now I understand.  

I just got my macOS VM running and trying to understand this environment. At least now I can help with macOS issues.

:)

**Comment by nikita-golovin** [comment]
The version installed via HomeBrew is still 6.0.2. While expected version is 6.0.4!

**Comment by nikita-golovin** [comment]
Seem to be fixed now (6.0.4 version was added 14 hours ago) in Homebrew repo

**Comment by MaximoTrinidad** [comment]
It's confirmed!

Executing the line below will install PScore 6.0.4 GA version now:
```
brew cask install powershell
```
![macos_pwshga_2018-08-14_8-17-09](https://user-images.githubusercontent.com/19892435/44091153-86c9005e-9f9a-11e8-9803-4fdb3ff6c4ad.jpg)
